util.go.ipfs

```diff
@@ -0,0 +1,292 @@
+package ipfs
```

Comment:
similar to previous comment, feel free to just change the package name, as the same file can be re-used under the TRON-US domain as a separate package